functions/_fzf_search_history.fish

```diff
@@ -1,6 +1,9 @@
 function _fzf_search_history --description "Search command history. Replace the command line with the selected command."
     # history merge incorporates history changes from other fish sessions
-    builtin history merge
+    # skip merging in private mode as it clears current session history
+    if test -n "$fish_provate_mode"
```

Comment:
oops just realized there's a typo here. I'll fix it

Reply:
Time to get a spell checker for vim I guess 😅